pkgs/applications/audio/baudline/default.nix

```diff
@@ -63,6 +63,7 @@ stdenv.mkDerivation rec {
     # See http://www.baudline.com/faq.html#licensing_terms.
```

Comment:
Can you share a bit of info how this change was generated? Particularly how were "packages where binary native code makes it from the downloaded src to the output" packages identified?

Reply:
A few methods.

A first pass to find candidates used a grep for either architecture names in urls or uses of `patchelf`. Then I would look manually at the source package(s) if I had any doubts and try and figure out if the binaries made it to the output.